dataprovider/src/main/java/cm/aptoide/pt/dataprovider/ws/v7/GetAppRequest.java

```diff
@@ -102,8 +112,8 @@ public static GetAppRequest of(long appId, String storeName,
 
   @Override
   protected Observable<GetApp> loadDataFromNetwork(Interfaces interfaces, boolean bypassCache) {
-    return interfaces.getApp(body, bypassCache,
-        appBundlesVisibilityManager.shouldEnableAppBundles());
+    return interfaces.getApp(bypassCache ? "no-cache" : "false",
```

Comment:
What does "false" mean in this context? I've looked for Cache-Control but I didn't see any meaning in what the "false" value means or what it does.

Reply:
Just wanted the parameter to be ignored when we don't want to bypass cache and can't find the default Cache-control header. Will look for an alternative to this

Reply:
ended up having to pass null in the request header so it is ignored by okhttp.